content/millennium/r4/other/binary.md

```diff
@@ -40,24 +40,50 @@ _Implementation Notes_
 
 <%= authorization_types(provider: true, patient: true, system: true) %>
 
-_Implementation Note_
+_Implementation Notes_
 
-The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentReference.read` scope is required.
+* For attachments with the `application/pdf` contentType, the `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentReference.read` scope is required.
+* For attachments with the `text/html` contentType, the `Binary.read` scope and the `DocumentReference.read` scope are required.
 
 ### Headers
 
+Either `DocumentReference.content.attachment.contentType` or `DiagnosticReport.presentedForm.contentType` should be used to set the `Accept` header. An `Accept` header of `application/fhir+json` could be supplied instead, if the JSON Binary resource is desired instead of the raw data.
+
+This resource will not accept the `application/json` mime type unless the underlying binary data is json.
+
 <%= headers fhir_json: true %>
 
+#### FHIR Accept Type
+
 ### Example
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197198634
+<%= headers head: {'Accept': 'application/fhir+json'} %>
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/TR-21961261
 
 #### Response
 
 <%= headers status: 200 %>
-<%= json(:r4_binary_entry) %>
+<%= json(:r4_binary_tr_json_entry) %>
+
+<%= disclaimer %>
+
+#### Native Accept Type
+
+### Example
```

Comment:
For FHIR Accept type and this one can we remove the 'Example' header and update the 'Native Accept Type' to something like 'Native Accept Example' please. I think this 'Example' header might result in a list of confusing reference links at the top of the page.

Reply:
Updated here: 726427a50d3998f20538bfe1bfcaecee2f847927